components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m

```diff
@@ -44,13 +51,13 @@
 static NSString *const kItemTitleLong2Arabic =
     @"وتم عل والقرى إتفاقية, عن هذا وباءت الغالي وفرنسا.";
 
-@interface MDCSnackbarManagerInternal ()
+@interface MDCSnackbarManagerInternal (TestInterface)
```

Comment:
Tip: Please name the category "MDCSnackbarMessageViewSnapshotTests" or something unique to avoid collisions.